android/app/build.gradle

```diff
@@ -148,8 +148,8 @@ android {
         applicationId "me.rainbow"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 211
-        versionName "1.9.23"
+        versionCode 213
```

Comment:
I had to do an extra bump here because we submitted a hotfix to Playstore that we never shipped, thus I needed to bump one more to submit bundle.